optax/contrib/momo.py

```diff
@@ -0,0 +1,245 @@
+# Copyright 2019 DeepMind Technologies Limited. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""MoMo.
+Implementation of
+"MoMo: Momentum Models for Adaptive Learning Rates" 
+(https://arxiv.org/abs/2305.07583) by Fabian Schaipp, Ruben Ohana,
+Michael Eickenberg, Aaron Defazio and Robert M. Gower.
+"""
+from typing import NamedTuple, Optional
+import chex
+import jax.numpy as jnp
+import jax.tree_util as tu
+from jax import Array
+from jax.lax import cond
```

Comment:
import modules rather than functions (i.e, use from jax import lax instead of the above). But FYI in this case one can access it as jax.lax.cond, without the import